systems/controllers/QPCommon.cpp

```diff
@@ -502,21 +502,24 @@ int setupAndSolveQP(NewQPControllerData *pdata, std::shared_ptr<drake::lcmt_qp_c
   
   // add in body spatial equality constraints
   // VectorXd body_vdot;
-  MatrixXd orig = MatrixXd::Zero(4,1);
-  orig(3,0) = 1;
+  Vector4d orig1 = Vector4d::Zero();
+  orig1(3) = 1.0;
```

Comment:
The old kinsol expects pts as 4x1 vector, with the last entry being 1. The new kinsol only uses 3x1 vector, so I need both.